ecommerce-ui/src/App.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { Component } from 'react';
+import ReactDOM from 'react-dom';
+import './App.css';
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
+import TitleBarGridList from './Properties';
+import ShoppingCartPanel from './ShoppingCart';
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.handleAddedToCart = this.handleAddedToCart.bind(this);
+    this.state = {propertyAdded: []}
+  }
+
+  handleAddedToCart(property) {
+    console.log(property);
+    this.setState({propertyAdded: property});
```

Comment:
You're adding a property, but it should be appended to the array in state. Think of how you can retrieve the `propertyAdded` array from previous state, then append the new property and set it to state.